rcode-reader/src/main/java/eu/rssw/pct/IParameter.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Decompiled with CFR 0_120.
+ */
+package eu.rssw.pct;
+
+public interface IParameter {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1214)